event/event.go

```diff
@@ -154,7 +155,7 @@ func NewEvent(ctx context.Context, ef store.EventFile, hub lab.Hub) (Event, erro
 		closers:       []io.Closer{ctf, guac, hub, keyLoggerPool},
 		dockerHost:    dockerHost,
 		keyLoggerPool: keyLoggerPool,
-		teamQueue:     make(chan *store.Team, conf.Capacity),
+		teamQueue:     make(chan store.Team, conf.Capacity),
```

Comment:
Is this safe? And why change it? Are we sure no pass-by-value problems will occur due to this change?

Reply:
Yes pointer would have been a lot better. I thought it was okay because the all the store.Team functions return store.Team as a value and not a pointer.

The reason i changes it was because then we loop `ev.store.GetTeams` and use a pointer, the underlining memory will change when the loop runs, and thus making all the queued teams the same.

I'm very welcome to feedback on how to solve this.